engine/src/main/java/org/terasology/rendering/cameras/OculusStereoCamera.java

```diff
@@ -172,6 +172,9 @@ public Matrix4f getInverseViewProjectionMatrix() {
         return null;
     }
 
+    /**
+     * @deprecated
```

Comment:
I would like to know this, too.  Unfortunately, the codebase doesn't seem to explain these deprecations very well.  I think I'm going to file a BR for these deprecations, because not everyone looks at CheckStyle results, anyway.